skins/oasis/modules/PageHeaderController.class.php

```diff
@@ -138,6 +138,9 @@ protected function getDropdownActions() {
 		$actions = array_merge( $editActions,
 			array( 'history', 'move', 'protect', 'unprotect', 'delete', 'undelete', 'replace-file' ) );
 
+		// Enable to modify actions list on dropdown
+		wfRunHooks( 'DropdownActions', [ &$actions ] );
```

Comment:
I think that you should be more specific in the hook's name. What about `ArticleViewDropdownActions`?